src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
+
+  @GetMapping("/greeting")
+  public String greeting(
```

Comment:
For future Pull Requests, I recommend using 4 spaces of indentation instead of 2. Check out the [Save actions plugin](https://github.com/Wolox/java-bootstrap#intellij-save-action-plugin), which facilitates some style changes 😄

Reply:
I actually love to use two, but if here we tend to use 4, I don't have really a problem. I'm changing it on every code I write for now on.